targets/trusted_root.json

```diff
@@ -44,10 +44,10 @@
       "certChain": {
         "certificates": [
           {
-            "rawBytes": "MIIB9zCCAXygAwIBAgIUALZNAPFdxHPwjeDloDwyYChAO/4wCgYIKoZIzj0EAwMwKjEVMBMGA1UEChMMc2lnc3RvcmUuZGV2MREwDwYDVQQDEwhzaWdzdG9yZTAeFw0yMTEwMDcxMzU2NTlaFw0zMTEwMDUxMzU2NThaMCoxFTATBgNVBAoTDHNpZ3N0b3JlLmRldjERMA8GA1UEAxMIc2lnc3RvcmUwdjAQBgcqhkjOPQIBBgUrgQQAIgNiAAT7XeFT4rb3PQGwS4IajtLk3/OlnpgangaBclYpsYBr5i+4ynB07ceb3LP0OIOZdxexX69c5iVuyJRQ+Hz05yi+UF3uBWAlHpiS5sh0+H2GHE7SXrk1EC5m1Tr19L9gg92jYzBhMA4GA1UdDwEB/wQEAwIBBjAPBgNVHRMBAf8EBTADAQH/MB0GA1UdDgQWBBRYwB5fkUWlZql6zJChkyLQKsXF+jAfBgNVHSMEGDAWgBRYwB5fkUWlZql6zJChkyLQKsXF+jAKBggqhkjOPQQDAwNpADBmAjEAj1nHeXZp+13NWBNa+EDsDP8G1WWg1tCMWP/WHPqpaVo0jhsweNFZgSs0eE7wYI4qAjEA2WB9ot98sIkoF3vZYdd3/VtWB5b9TNMea7Ix/stJ5TfcLLeABLE4BNJOsQ4vnBHJ"
```

Comment:
FYI @woodruffw the order of the certs was reversed and we're fixing it in this upcoming root signing to go out this week. Have you released a sigstore-python with the trusted root support yet? If so you may need a patch fix

Reply:
@bdehamer will this break verification in sigstore-js as used in npm CLI? 😬

Reply:
I believe this shouldn't cause issues for `sigstore-python` either (since we punt all chain order/verification down to pyOpenSSL's `X509Store`, which shouldn't care about the order of insertion), but CC @tetsuo-cpp and @tnytown to confirm.

Reply:
Woooh!! Robustness of both of the client libs :D Love to hear it.

Reply:
Great! And we'll fix this for staging once the ceremony is done.

Reply:
Yep, I also think `sigstore-python` should handle this fine.